GauchoGrub/app/src/main/java/edu/ucsb/cs/cs190i/gauchogrub/gauchogrub/FavoriteRecyclerAdapter.java

```diff
@@ -0,0 +1,140 @@
+package edu.ucsb.cs.cs190i.gauchogrub.gauchogrub;
+
+
+import android.content.Context;
+import android.media.Image;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import com.daimajia.swipe.SwipeLayout;
+
+import edu.ucsb.cs.cs190i.gauchogrub.gauchogrub.db.models.DiningCommon;
+import edu.ucsb.cs.cs190i.gauchogrub.gauchogrub.db.models.Favorite;
+import edu.ucsb.cs.cs190i.gauchogrub.gauchogrub.db.models.MenuItem;
+import io.requery.Persistable;
+import io.requery.android.QueryRecyclerAdapter;
+import io.requery.query.Result;
+import io.requery.sql.EntityDataStore;
+
+public class FavoriteRecyclerAdapter extends QueryRecyclerAdapter<MenuItem, FavoriteRecyclerAdapter.ViewHolder> {
+
+    private EntityDataStore<Persistable> dataStore;
+    private String diningCommon;
+    private int diningCommonId;
+    private Context context;
+    private FavoriteRecyclerAdapter thisAdapter;
+
+    protected FavoriteRecyclerAdapter(String diningCommon, Context context) {
+        super(MenuItem.$TYPE);
+        dataStore = ((GGApp) context.getApplicationContext()).getData();
+        this.diningCommon = diningCommon;
+        this.diningCommonId = dataStore.select(DiningCommon.class).where(DiningCommon.NAME.eq(diningCommon)).get().first().getId();
```

Comment:
Each segment in long queries like this one should go on a new line for readability.
`dataStore`
`.select(...)`
`.where(...)`
`.get(...)`
`.first(...)`